gormapplication/application.go

```diff
@@ -97,6 +99,14 @@ func (g *GormBase) InvitationRepository() invitation.InvitationRepository {
 	return invitation.NewInvitationRepository(g.db)
 }
 
+func (g *GormBase) InvitationService() invitationservice.InvitationService {
+	return invitationservice.NewInvitationService(g)
+}
+
+func (g *GormBase) PermissionService() permissionservice.PermissionService {
+	return permissionservice.NewPermissionService(g.db)
+}
```

Comment:
Both method should return Model Services (need a better name). Not Controller Services.
Model Services is purely a DB layer.
Controller Services is not. They may use DB layer (if they do they should provide transaction control). They may call remote services.

Reply:
I've renamed these for now to put "Model" back into the name.